src/util.h

```diff
@@ -56,12 +57,14 @@ extern std::atomic<bool> fReopenDebugLog;
 extern CTranslationInterface translationInterface;
 
 /**
- * Translation function: Call Translate signal on UI interface, which returns a boost::optional result.
+ * Translation function: Call Translate signal on UI interface, which returns a std::optional result.
  * If no translation slot is registered, nothing is returned, and simply return the input.
  */
 inline std::string _(const char* psz)
 {
-    boost::optional<std::string> rv = translationInterface.Translate(psz);
+    std::optional<std::string> rv;
+    boost::optional<std::string> tmp = translationInterface.Translate(psz);
+    if (tmp) rv = tmp.get();
     return rv ? (*rv) : psz;
```

Comment:
Totally non-blocking comment: we could use `std::optional` member function `value_or()` to avoid the ternary operator on the return statement.

Reply:
Good idea, I implemented the suggested change, thanks